lib/rock/packaging/debian.rb

```diff
@@ -250,11 +250,7 @@ def update_osdeps_list(pkginfo, file, selected_platform)
                     pkg_name = pkginfo
                     release_name, is_osdep = native_dependency_name(pkg_name, selected_platform)
                     Packager.debug "Native dependency of ruby package: '#{pkg_name}' -- #{release_name}, is available as osdep: #{is_osdep}"
-                    if is_osdep
-                        dependency_debian_name = release_name
-                    else
-                        dependency_debian_name = debian_ruby_name(pkg_name)
-                    end
+                    dependency_debian_name = release_name
```

Comment:
To increase consistency does it make sense to retrieve the pkginfo when string is given, and always use the else branch?

Reply:
A string is only provided if no pkginfo could be found by the calling code, so this code would not be able to find a pkginfo either. That probably happens for ruby packages that are not tracked by autoproj and do not exist in the operating system, and only when requested explicitly.